### ISSUE TITLE
A option to export all Savs, from all games at once?

### ISSUE THREAD

**Issue by Conan179** [d3m3vilurr/vita-savemgr#84]
I have a Feature request.
Can you add a option to export all games, from all games at once?
greetz   ##Conan

**Comment by d3m3vilurr** [comment]
lol

**Comment by Conan179** [comment]
why "lol"?

**Comment by d3m3vilurr** [comment]
i hasn't thinking this idea before. cuz ui steps. but pretty useful if we can  choose right slot :)

**Comment by ghost** [comment]
Bruh

**Comment by ghost** [comment]
For this long it still hasn't been added

**Comment by ghost** [comment]
Some save data have encryption. So that's why we want this. For example Rabi-Ribi doesn't allow if i'm correct

**Comment by theAkito** [comment]
> For this long it still hasn't been added

You're right. When will you finally add it?! Please, open your pull request.

**Comment by ghost** [comment]
I can't tho

**Comment by ghost** [comment]
The dev has to

**Comment by ghost** [comment]
Unless I can code which I can't
